presentation/README.md

```diff
@@ -0,0 +1,433 @@
+    txtComposer = new TextField();
+    chkClassical.addActionListener(itemEvent -> {
+      txtComposer.setEditable(chkClassical.isSelected());
+      if (!chkClassical.isSelected()) {
+        txtComposer.setText("");
+      }
+    });
+    txtComposer.setSize(widthTxt, heightTxt);
+    txtComposer.setEditable(model.getIsClassical());
+
+    apply = new JButton("Apply");
+    apply.addMouseListener(new MouseAdapter() {
+      @Override
+      public void mouseClicked(final MouseEvent e) {
+        saveToPMod();
+        loadFromPMod();
+      }
+    });
+    cancel = new JButton("Cancel");
+    cancel.addMouseListener(new MouseAdapter() {
+      @Override
+      public void mouseClicked(final MouseEvent e) {
+        loadFromPMod();
+      }
+    });
+
+    b2.add(txtArtist);
+    b2.add(txtTitle);
+
+    b2.add(chkClassical);
+    b2.add(txtComposer);
+
+    b2.add(apply);
+    b2.add(cancel);
+
+    final var x = 200;
+    final var y = 200;
+    final var width = 500;
+    final var height = 300;
+    frame.setDefaultCloseOperation(javax.swing.WindowConstants.EXIT_ON_CLOSE);
+    frame.setBounds(x, y, width, height);
+    frame.setVisible(true);
+  }
+
+}
+
+```
+
+Class `Album` is to store information of a album.
+
+```java
+@Setter
+@Getter
+@AllArgsConstructor
+public class Album {
+  /**
+   * the title of the album.
+   */
+  private String title;
+  /**
+   * the artist name of the album.
+   */
+  private String artist;
+  /**
+   * is the album classical, true or false.
+   */
+  private boolean isClassical;
+  /**
+   * only when the album is classical,
+   * composer can have content.
+   */
+  private String composer;
+}
+
+```
+
+Class `DsAlbum` is store the information of all the albums that will be shown on GUI.
+
+```java
+/**
+ * a class used to deal with albums.
+ */
+@Slf4j
+@Getter
+public class DsAlbum {
+  /**
+   * albums a list of albums.
+   */
+  private final List<Album> albums;
+
+  /**
+   * a constructor method.
+   */
+  public DsAlbum() {
+    this.albums = new ArrayList<>();
+  }
+
+  /**
+   * a method used to add a new album to album list.
+   *
+   * @param title       the title of the album.
+   * @param artist      the artist name of the album.
+   * @param isClassical is the album classical, true or false.
+   * @param composer    only when the album is classical,
+   *                    composer can have content.
+   */
+  public void addAlbums(final String title,
+                        final String artist, final boolean isClassical,
+                        final String composer) {
+    if (isClassical) {
+      this.albums.add(new Album(title, artist, true, composer));
+    } else {
+      this.albums.add(new Album(title, artist, false, ""));
+    }
+  }
+}
+
+```
+
+ Class `PresentationMod` is used to control all the action of GUI.
+
+```java
+/**
+ * The class between view and albums, it is used to control the data.
+ */
+@Slf4j
+public class PresentationMod {
```

Comment:
Could we call it PresentationModel instead of PresentationMod? I think the mod has another meaning.